wiki/Ranking_criteria/Simplified_ranking_criteria/en.md

```diff
@@ -0,0 +1,197 @@
+
+### Spread
+
+- **Difficulty names must have progression.** 
+  - Easy -> Normal -> Hard -> Insane -> Expert is default.
+  - Logical naming schemes like Seed -> Sprout -> Tree are okay too.
+  - **Exception:** The hardest difficulty can use a custom name, like Normal -> Hard -> *Melancholy*
+- **Don't skip difficulties.** For example, if you have Normal and Insane difficulties, you need a Hard too.
+- **A [guest mapper](/wiki/Beatmap/Guest_difficulty) can't create more difficulties than the [host](/wiki/Beatmap/Beatmap_host).**
+- **Minimum requirement for lowest difficulty by mode and length:**
+
+|  | osu! | osu!taiko |
+| --: | :-: | :-: |
+| **0:30 to 3:30** | Normal | Futsuu |
+| **3:30 to 4:15** | Hard | Muzukashii |
+| **4:15 to 5:00** | Insane | Oni |
+
+|  | osu!catch | osu!mania |
+| --: | :-: | :-: |
+| **0:30 to 2:30** | Salad | Normal |
+| **2:30 to 3:15** | Platter | Hard |
+| **3:15 to 4:00** | Rain | Insane |
+
+### Hitsounds
+
+- **Maps must be [hitsounded](/wiki/Beatmapping/Hitsound),** excluding osu!mania maps.
+- **Every clickable object must have audible feedback.**
+
+### Timing
+
+::: Infobox
+![](./img/2b.png "Two objects on the same tick")
+:::
+
+- **Maps must be correctly timed.** This includes BPM and time signatures.
+- **All difficulties must use the same timing.**
+- **Do not change timing to adjust slider velocity.**
+- **Objects must be snapped to timeline ticks.**
+- **Only one object is allowed per tick,** excluding osu!mania maps.
+
+## Metadata
+
+- **Metadata must be accurate.**
+  - Use a [primary metadata source](/wiki/Beatmap/Primary_metadata_source).
+  - If the song has a Ranked or Loved map, use that map's metadata unless it's blatantly wrong.
+- **Use [Modified Hepburn romanisation](https://en.wikipedia.org/wiki/Hepburn_romanization#Features) for Japanese words.**
+
+### Tags
+
+  - **Add usernames of anyone who contributes to the map.** This doesn't include modders.
+  - **Add the song's [genre and language](/wiki/Beatmap/Genre_and_language).**
+  - **Add `featured artist` if the song is in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).**
+  - **Other tags must be relevant to the song/map.**
+
+### Title
+
+  - **Songs used for TV must have `(TV Size)`.**
+  - **Replace game version markers with `(Game Ver.)`.**
+  - **Replace short version markers with `(Short Ver.)`.**
+  - **Use `(Cut Ver.)` to indicate an unofficial song cut.**
+  - **Use `(Extended Edit)` to indicate an unofficial song extension.**
+  - **Use `(Sped Up Ver.)` to indicate an unofficial song tempo increase.** For some genres, `(Nightcore Mix)` is an okay alternative.
+
+### Source
+
+- **Use the Source field if the song is from another media source, like a game or a movie.**
+
+## Files
+
+- **Don't use inappropriate songs or backgrounds.** See [song content rules](/wiki/Rules/Song_content_rules) and [visual content considerations](/wiki/Rules/Visual_content_considerations).
```

Comment:
expand for videos and potentially other inappropriate metadata? usually difficulty names and tags when talking about ranking, as (assuming they are correct) song/artist/source names are covered.